### ISSUE TITLE
CEF Crash 0x01978697

### ISSUE THREAD

**Issue by PlatinMTA** [multitheftauto/mtasa-blue#3705]
### Describe the bug

CEF crashes since r22708

```
Version = 1.6-release-22712.0.000
Time = Sat Sep  7 16:43:54 2024
Module = D:\MTA San Andreas 1.6\MTA\libcef.dll
Code = 0x80000003
Offset = 0x01978697

EAX=15BEF4E0  EBX=15BEF4D4  ECX=63D7C492  EDX=00000000  ESI=27180A1C
EDI=15BEF4E0  EBP=15BEF500  ESP=15BEF4D4  EIP=52CF8697  FLG=00000246
CS=0023   DS=002B  SS=002B  ES=002B   FS=0053  GS=002B
```

### Steps to reproduce

1. Open MTA
2. Go to Config
3. Go to web browser
4. Crash

### Version

Client: r22708

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by gta191977649** [comment]
Yeah Same issue.
![image](https://github.com/user-attachments/assets/84b56df2-ded0-42c1-a908-6014bd77028f)

**Comment by thisdp** [comment]
focusBrowser crashes, this issue affects dgs
requestBrowserDomains({"123.com"},false,function() end) can also crash MTA

**Comment by Xenius97** [comment]
Only latest nightly is crashing, stable versions working well. They know about this issue.

**Comment by TheNormalnij** [comment]
Fixed in 71f92d55c18a0a191eae7f7ea77bfeab89e9aab2
